wave_vision/include/wave/vision/fast_detector.hpp

```diff
@@ -94,9 +98,9 @@ class FASTDetector : public FeatureDetector {
     *  @param check_config, the FASTParams struct containing the desired
     *  configuration values.
     */
-    void checkConfiguration(FASTParams& check_config);
+    void checkConfiguration(const FASTParams& check_config);
 };
 
-} // end of namespace wave
+}  // end of namespace wave
```

Comment:
Yes it's only purpose is enforcing google style guide. We're not enforcing it in this project right? I just happened to have it on in my IDE.